ui/pages/SummaryData/components/SampleMetadata.jsx

```diff
@@ -124,68 +96,19 @@ const getColumns = (data) => {
     ...[...Array(maxSavedVariants).keys()].map(i => VARIANT_COLUMNS.map(
       col => ({ name: `${col}-${i + 1}`, secondaryExportColumn: col === GENE_COL ? `gene_id-${i + 1}` : null }),
     )),
-  ).map(({ name, ...props }) => ({ name, content: name, ...props }))
-}
-
-export const SampleMetadata = React.memo(({ projectGuid, queryForm, data, user }) => (
-  <div>
-    <InlineHeader size="medium" content="Project:" />
-    <AwesomeBar
-      categories={SEARCH_CATEGORIES}
-      placeholder="Enter project name"
-      inputwidth="350px"
-      getResultHref={getResultHref}
-    />
-    {(user.isAnalyst ? ANALYST_VIEW_ALL_PAGES : VIEW_ALL_PAGES).map(({ name, path }) => (
-      <span key={path}>
-        &nbsp; or &nbsp;
-        <ActiveDisabledNavLink to={`/summary_data/sample_metadata/${path}`}>{`view all ${name} projects`}</ActiveDisabledNavLink>
-      </span>
-    ))}
-    <HorizontalSpacer width={20} />
-    {queryForm}
-    <DataTable
-      striped
-      collapsing
-      horizontalScroll
-      downloadFileName={`${FILENAME_LOOKUP[projectGuid] || (data?.length && data[0].project_id.replace(/ /g, '_'))}_${new Date().toISOString().slice(0, 10)}_Metadata`}
-      idField="subject_id"
-      defaultSortColumn="family_id"
-      emptyContent={projectGuid ? '0 cases found' : 'Select a project to view data'}
-      data={data}
-      columns={getColumns(data)}
-      rowsPerPage={100}
-    />
-  </div>
-))
-
-SampleMetadata.propTypes = {
-  data: PropTypes.arrayOf(PropTypes.object),
-  projectGuid: PropTypes.string,
-  queryForm: PropTypes.node,
-  user: PropTypes.object,
+  )
 }
 
-const parseResponse = ({ rows }) => ({ data: rows })
-
-const LoadedSampleMetadata = ({ match, user }) => (
-  <StateDataLoader
-    url={match.params.projectGuid ? `/api/summary_data/sample_metadata/${match.params.projectGuid}` : ''}
-    parseResponse={parseResponse}
-    queryFields={(user.isAnalyst && match.params.projectGuid !== ALL_PROJECTS_PATH) ? AIRTABLE_FIELDS : FIELDS}
-    childComponent={SampleMetadata}
-    projectGuid={match.params.projectGuid}
-    user={user}
-  />
-)
-
-LoadedSampleMetadata.propTypes = {
-  match: PropTypes.object,
-  user: PropTypes.object,
+const mapStateToProps = (state, ownProps) => {
```

Comment:
this feels like it could be a silly question, but why can't you just export a component that returns `LoadReportTable`?

Reply:
I'm not sure I understand the question. At the bottom of the file we export a component that binds these props to `LoadReportTable`, resulting in a Report table with all the desired props being exported
```
export default connect(mapStateToProps)(LoadReportTable)
```

Reply:
I guess I meant why use `mapStateToProps` instead of exporting the component like it's done [here](https://github.com/broadinstitute/seqr/pull/3785/files/aa0f3bc81387b0dcd663854d92052e18481eecd0#diff-94ee2491f4eb1a73f4d1de4a532827b7c7b785059d3023f2cb1c6ab9588c935bR33), but I just realized that the global state is accessed to get the user so that is probably why

Reply:
yeah, thats exactly it, thsi componenet needs the `user` while that other example does not